### DISCUSSION TITLE
How to add internationalization support for native system menu on MacOS?

### DISCUSSION THREAD

**Discussion by percy507** [tauri-apps/tauri#4985] [Q&A] [answered]
My macbook-pro's region is China and I have checked the webview inside tauri frame, the `navigator.language` is `zh-cn`. 

But in the menubar, some native menus's label is English. I wonder why that is.

```rs
let edit_menu = Menu::new()
    .add_native_item(MenuItem::Undo)
    .add_native_item(MenuItem::Redo)
    .add_native_item(MenuItem::Separator)
    .add_native_item(MenuItem::Copy)
    .add_native_item(MenuItem::Cut)
    .add_native_item(MenuItem::Paste)
    .add_native_item(MenuItem::Separator)
    .add_native_item(MenuItem::SelectAll);
```

**Comment by FabianLars** [comment, marked as answer]
Even though they are called "native" we still have to implement them, or at least internationalization, ourselves. We won't fix this with the current menu implementation but with the new menu crate [muda](https://github.com/amrbashir/muda) (wip).

**Reply by percy507** [reply]
Thanks. I will wait.

**Comment by romainbriand** [comment]
Hi! Do you have any ETA on this topic? Feels like it is the same with "native" dialogs (they stay in English whatever the app/system locale). I'm quite a newbie with Tauri but it feels like a blocker for complete apps internationalization.
